shared-bindings/bitmaptools/__init__.h

```diff
@@ -88,4 +89,21 @@ void common_hal_bitmaptools_dither(displayio_bitmap_t *dest_bitmap, displayio_bi
 void common_hal_bitmaptools_alphablend(displayio_bitmap_t *destination, displayio_bitmap_t *source1, displayio_bitmap_t *source2, displayio_colorspace_t colorspace, mp_float_t factor1, mp_float_t factor2,
     bitmaptools_blendmode_t blendmode, uint32_t skip_source1_index, bool skip_source1_index_none, uint32_t skip_source2_index, bool skip_source2_index_none);
 
+typedef struct {
+    union {
+        struct {
+            int16_t x1, y1, x2, y2;
+        };
+        int16_t arr[4];
+    };
+} bitmaptools_rect_t;
+
+#define ARGS_COORD_PAIR ARG_x1, ARG_x2, ARG_x3, ARG_x4
```

Comment:
Did you mean?
```suggestion
#define ARGS_COORD_PAIR ARG_x1, ARG_y1, ARG_x2, ARG_y2
```
I think I would prefer `ARGS_COORD_PAIR` to be `ARGS_X1_Y1_X2_Y2` or similar so it is more self-evident.